Do-Good-Get-Good/do-good-get-good
# Feature/show all activities for superadmin

Superadmins are now able to add chat posts for all activities.

#### How to test
1. Log in as superadmin
2. Go to chat page
3. Add a post and make sure all activities show up
4. Now do the same but as a normal user or admin and make sure only connected activities show up.
